content/national-qualifications-framework/ncba/assessments/ncba-sa4-design-thinking-empathise-feedback-data/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+content_type: project
+flavours:
+- none
+prerequisites:
+  hard:
+  - national-qualifications-framework/ncba/content/design-thinking-empathise
+ready: true
+submission_type: link
+tags: 
+ - docx
+ - ncba
+ - design-thinking
+ - empathise
+title: NCBA SA4 Design Thinking - Empathise Feedback Data
+---
+
+Please do the following:
+
+1. Follow [this link](https://docs.google.com/document/d/1NI2g3uT4Pk19iB2b82kSwef6003sB_mP/edit?usp=sharing&ouid=106698657596806218419&rtpof=true&sd=true) and download the document. If you have an Umuzi email address then you will have access to the document. If you do not have an Umuzi email address you may need to request access.
+
+2. Complete the assessment. The instructions are in the document. 
+   
+3. Upload the document to google drive and submit a link. Ensure that the submission has the `docx` file extension. Please make sure that the link works. If your work is private then we won't be able to review it. You can double-check this by opening your own link in an incognito browser.  
+
+4. Your link should look like this:
```

Comment:
Maybe say "your link would look something like this:"

It's good to be explicit about the fact that this is not exact.